app/components/questionnaire/job-name.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const JobName = Ember.Component.extend({
+  tagName: 'div',
```

Comment:
I think the default tag for components is div, and you have a div in the hbs too. Maybe that's causing the display bugs. Perhaps try removing the outer div in the hbs and add [classNames](https://guides.emberjs.com/v2.13.0/components/customizing-a-components-element/#toc_customizing-the-element-s-class) here?